src/app/_components/NotificationDialog.tsx

```diff
@@ -0,0 +1,56 @@
+import { Dialog, DialogPanel, DialogTitle } from '@headlessui/react'
+import { CheckCircle, XCircle } from '@phosphor-icons/react'
+import { X } from '@phosphor-icons/react/dist/ssr'
+
+import { Icon } from '@/components/Icon'
+
+export type StatusType = 'success' | 'warning'
+
+type NotificationDialogProps = {
+  isOpen: boolean
+  setIsOpen: (arg: boolean) => void
+  status?: StatusType
+}
+
+const dialogStatus = {
+  success: {
+    title: 'Successfully subscribed!',
```

Comment:
I feel like `title` - the message of the toast - should be passed in as a prop intead of being hardcoded.

In theory, a success or error toast could be used in other contexts. The ecosystem project form might use it for instance :)

Reply:
I see what you mean, but I feel like it's easier to have an overview of statuses in the component in the case we want to change the copy or structure, we have it in one place. We can be more specific and have success-subscription, success-registration if there's ever a need. @mirhamasala any thoughts?

Reply:
Agree with @CharlyMartin on this one.